server/meshmodel/aws-cloudfront-controller/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,58 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "name": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
```

Comment:
This is good. Can we swap out this example with a more commonplace example (e.g. Kubernetes Namespace and Kubernetes Pod), if only to help ensure users understand the relationship type easily?

Reply:
Sure. I'll update PR.